### DISCUSSION TITLE
How to change TextField's UNDERLINE color.

### DISCUSSION THREAD

**Discussion by MatsuSpring** [flet-dev/flet#4136] [Q&A] [answered]
### Question

I'm trying to change the color of the underline in a `TextField` control where the` border` property is set to `ft.InputBorder.UNDERLINE`.
However, even when I try to change the color using the `border_color` property as shown in the code below, the underline color remains the same.

<img width="338" alt="image" src="https://github.com/user-attachments/assets/10e3286c-6597-4b1e-b46f-eb26a1f4b12e">

Is this a known limitation of Flet, or am I missing something?
Does anyone have any suggestions or solutions?

(Apologies if the English translation is a bit awkward, as it was generated by Gemini.)

### Code sample

```python
import flet as ft

def main(page: ft.Page):
    tf = ft.TextField(
        label="Input Name",
        border=ft.InputBorder.UNDERLINE,
        border_color=ft.colors.ORANGE
    )

    page.add(tf)

ft.app(main)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ClearSafety** [comment]
The property used to change the border color is 'border_color' that seems to be incompatible with the property 'border'.
I couldn't find a way to change the underline border color. 
The only way to change its color is using the property 'focused_border_color', but it works only when the field is focused. 

I suggest you to raise an Issue

**Comment by ndonkoHenri** [comment, marked as answer]
Created an issue: https://github.com/flet-dev/flet/issues/4140
